src/internal.c

```diff
@@ -25561,6 +25567,22 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
                                 }
                             }
 
+                            if ((ret = wc_curve25519_check_public(
+                                    input + args->idx, args->length,
+                                    EC25519_LITTLE_ENDIAN)) != 0) {
+                        #ifdef OPENSSL_EXTRA
```

Comment:
Let's put verbose alerts in a separate define that OPENSSL_EXTRA may turn on, if that seems appropriate.

Reply:
I will make the change: WOLFSSL_EXTRA_ALRETS